rust/lance-index/src/vector/ivf/transform.rs

```diff
@@ -31,28 +30,22 @@ use super::PART_ID_COLUMN;
 /// this transform is a Noop.
 ///
 #[derive(Debug)]
-pub struct IvfTransformer<T: ArrowFloatType>
-where
-    T::Native: Dot + L2,
-{
-    centroids: MatrixView<T>,
-    metric_type: MetricType,
+pub struct IvfTransformer {
+    centroids: Arc<FixedSizeListArray>,
```

Comment:
do we need wrap the array inside `Arc`?
this array type stores data in `ArrayRef` (which is inside `Arc` already)

Reply:
Not sure what is suggest? Could you elaborate a bit? 
Or maybe show an example in code?